onnxruntime/core/providers/openvino/backend_utils.cc

```diff
@@ -68,6 +66,20 @@ CreateCNNNetwork(const Provider_ModelProto& model_proto, const GlobalContext& gl
   } catch (...) {
     ORT_THROW(log_tag + "[OpenVINO-EP] Unknown exception while importing model to nGraph Func");
   }
+#else
+  InferenceEngine::CNNNetwork cnn_network;
+  const std::string model = model_proto.SerializeAsString();
+  InferenceEngine::Blob::Ptr blob = {nullptr};
+  try {
+    cnn_network = global_context.ie_core.ReadNetwork(model, blob);
+    LOGS_DEFAULT(INFO) << "Read network Done";
+  } catch (const std::exception& exp) {
```

Comment:
Aren't we supposed to use InferenceEngine::details::InferenceEngineException& e here ?